admin/app/views/workarea/admin/tax_rates/index.html.haml

```diff
@@ -56,9 +56,9 @@
               %td.align-center= rate.country.try(&:alpha2) || '-'
               %td.align-center= rate.region || '-'
               %td.align-center= rate.postal_code.presence || '-'
-              %td.align-center= number_to_percentage(rate.country_percentage * 100, precision: 1)
-              %td.align-center= number_to_percentage(rate.region_percentage * 100, precision: 1)
-              %td.align-center= number_to_percentage(rate.postal_code_percentage * 100, precision: 1)
+              %td.align-center= number_to_percentage(rate.country_percentage * 100, precision: 3, strip_insignificant_zeros: true)
```

Comment:
This is actually the default precision, but perhaps this should be configurable? It's the only real limiting factor that would potentially block you from seeing the exact input you entered in the form. Entering in a number like `7.0005` would result in the following admin UI:

<img width="344" alt="Screen Shot 2020-09-22 at 10 21 24 AM" src="https://user-images.githubusercontent.com/113026/93895039-6020c400-fcbd-11ea-93d7-a00f90ab65f0.png">

This is a bit strange to me, so I was thinking this could be an admin-configurable setting?